docs/topics/Disaster-Recovery.md

```diff
@@ -33,36 +47,78 @@ docker exec [container] pg_dump \
 
 > Cleaning the database is strongly recommended before overwriting it.
```

Comment:
Honestly let's make this an affirmative requirement before importing. I always clean before reimporting and failing to do so would probably result in broken imports. Bonus points if you convert all of these steps into a `procedure`, though I'm not sure whether Writerside will even let us do that.